src/util/CMakeLists.txt

```diff
@@ -42,12 +42,15 @@ target_sources(
 
 # This must be above the target_link_libraries call otherwise backtrace doesn't work
 if ("${san}" STREQUAL "")
-  target_link_libraries(clio_util PUBLIC Boost::stacktrace_backtrace dl libbacktrace::libbacktrace)
+  target_link_libraries(clio_util PUBLIC Boost::stacktrace_backtrace)
 endif ()
 
 target_link_libraries(
   clio_util
   PUBLIC Boost::headers
+         Boost::iostreams
+         Boost::log
+         Boost::log_setup
```

Comment:
I thought `Boost::headers` includes all the header only libraries

Reply:
It does, these are not header-only